### ISSUE TITLE
`npm audit` have some error

### ISSUE THREAD

**Issue by fupengl** [stylelint/stylelint#5429]
> Clearly describe the documentation issue

e.g. "The description of the `ignoreFiles` option is confusing ..."

> What solution would you like to see?

e.g. "I suggest rewriting it as ..."

![image](https://user-images.githubusercontent.com/20211964/127294727-b1f80ee6-1737-45cd-8d29-38901086f25b.png)

```json
    "stylelint": "^13.13.1",
    "stylelint-config-css-modules": "^2.0.0",
    "stylelint-config-prettier": "^8.0.2",
    "stylelint-config-rational-order": "^0.0.3",
    "stylelint-config-standard": "^20.0.0",
    "stylelint-declaration-block-no-ignored-properties": "^2.4.0",
    "stylelint-order": "^3.0.0"
```

**Comment by jeddy3** [comment]
Will be [fixed in `14.0.0`](https://github.com/stylelint/stylelint/issues/5205).

Please consider [contributing](https://stylelint.io/contributing) if you have time.

**Comment by fupengl** [comment]
> Will be [fixed in `14.0.0`](https://github.com/stylelint/stylelint/issues/5205).
> 
> Please consider [contributing](https://stylelint.io/contributing) if you have time.

Thanks
